src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -134,13 +134,13 @@ export class RetireJsEngine extends AbstractRuleEngine {
 		return Promise.resolve(retireJsCatalog);
 	}
 
-	/* eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars */
+	/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
 	shouldEngineRun(ruleGroups: RuleGroup[], rules: Rule[], target: RuleTarget[], engineOptions: Map<string, string>): boolean {
 		// If the engine was not filtered out, no reason to not run it
 		return true;
 	}
 
-	/* eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars */
+	/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
Typescript rule overrides the vanilla rule, so we only need to disable the former.